web/src/components/settings-page/api-key/index.tsx

```diff
@@ -166,6 +174,7 @@ export const APIKeyPage: FC = memo(function APIKeyPage() {
         open={Boolean(anchorEl)}
         anchorEl={anchorEl}
         onClose={handleCloseMenu}
+        PaperProps={menuStyle}
```

Comment:
could you attach a screenshot to show what this change makes to the UI?  👀

Reply:
fixed the description. 🙏

Reply:
I see, to set the width of the menu, should we use `className` instead? and besides, looks like it's redundant to set the maxHeight as well, so I guess it's better to remove unneeded code 🤔

Reply:
In this case, `className`doesn't work because we need to override the underlying paper class.
If you don't wanna use `PaperProps`, can use `classes={{paper: classes.menu}}` instead of that/

Reply:
got your point, then lets use PaperProps and remove unneeded maxHeight, wdyt? 👀